ui/src/shared/utils/useVisDomainSettings.test.ts

```diff
@@ -72,24 +71,158 @@ describe('getRemainingRange', () => {
     }
     expect(getRemainingRange([], timeRange, [null, null])).toEqual(null)
   })
-  it("should return the min y-axis if it's set", () => {
+  describe('should return a valid number from the stored domain as the min y-axis', () => {
```

Comment:
nit here but can we please have a newline between each `describe` and `it` block?